### PR TITLE
GraphQL quickstart

### PR DESCRIPTION
**Pull request by gloomweaver**

### CODE REVIEW

File: graphql/spicepod.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+kind: Spicepod
+name: graphql_quickstart
+datasets:
+  - from: graphql:https://api.github.com/graphql
+    name: repos
+    params:
+      auth_token: [auth_token]
+      json_path: data.viewer.repositories.nodes
+      query: |
+        {
```
Comment (digadeesh) [inline graphql/spicepod.yaml]: Does this work with both query {} and without?  If so, let's note that in this quickstart.

File: graphql/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# GraphQL Data Connector
+
+Follow these steps to get started with GraphQL as a Data Connector.
+With quickstart uses Github's GraphQL API as an example.
```
Comment (digadeesh) [inline graphql/README.md]: Quickstarts are separate from samples.  The scenario we are unlocking with quickstarts is: If you already have a graphql API you'd like to use with spice, you can use this to quickly get started.

Comment (digadeesh) [reply graphql/README.md]: So we can make this example a sample.  And I like the idea.  So lets move this to samples and make a proper quickstart.

File: graphql/spicepod.yaml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+                watchers (first: 10) {
+                  nodes {
+                    name
+                  }
+                }
+              }
+            }
+          }
+        }
+    description: eth recent blocks
```
Comment (digadeesh) [inline graphql/spicepod.yaml]: Since this is accelerating by default, describe that in the summary text above.

File: graphql/README.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+**Step 1** Generate a personal access token from Github. Follow the instructions [here](https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/creating-a-personal-access-token).
+
+**Step 2** Edit the `spicepod.yaml` file in this directory and replace `[auth_token]` with the personal access token generated in step 1.
+
+Follow the [quickstart guide](https://docs.spiceai.org/getting-started) to get started with the Spice.ai runtime.
+
+See the [datasets reference](https://docs.spiceai.org/reference/spicepod/datasets) for more dataset configuration options.
+
+To securely store your Clickhouse password, see [Secret Stores](https://docs.spiceai.org/secret-stores)
```
Comment (gloomweaver) [inline graphql/README.md]: Ops

File: graphql/spicepod.yaml (hunk 1 above)
Comment (gloomweaver) [reply graphql/spicepod.yaml]: It's works the same way as in any other graphql interface

File: graphql/README.md (hunk 4 above)
Comment (digadeesh) [inline graphql/README.md]:
```suggestion
To securely store your GraphQL API password, see [Secret Stores](https://docs.spiceai.org/secret-stores)
```

**Review by digadeesh — APPROVED** [review]

**Pull request merged**